gradle.properties

```
@@ -79,4 +79,8 @@ targetSdkVersion=36
 # Roborazzi
 roborazzi.test.record=false
 roborazzi.test.compare=false
-roborazzi.test.verify=false
\ No newline at end of file
+roborazzi.test.verify=false
+
+# Enable Gradle to auto-provision JDK 21
+org.gradle.java.installations.auto-detect=true
+org.gradle.java.installations.auto-download=true
```

Lets remove this line as I would rather the build fail if the SDK is not found. That would allow us to identify the issue and address it.